packages/flutter/test/widgets/scroll_controller_test.dart

```diff
@@ -226,7 +226,8 @@ void main() {
       ],
     ));
 
-    expect(() => controller.jumpTo(1.0), throwsAssertionError);
-    expect(() => controller.animateTo(1.0, duration: const Duration(seconds: 1), curve: Curves.linear), throwsAssertionError);
+    controller.jumpTo(1.0);
+    controller.animateTo(1.0, duration: const Duration(seconds: 1), curve: Curves.linear);
+    await tester.pumpUntilNoTransientCallbacks();
```

Comment:
add a pump above this one, because of https://github.com/flutter/flutter/issues/8186